jenkins/internal-ci/generic/branch/release.groovy

```diff
@@ -275,10 +275,10 @@ pipeline {
                         def build_cortx_images = build job: 'cortx-docker-images', wait: true,
                                     parameters: [
                                         string(name: 'CORTX_RE_URL', value: "https://github.com/shailesh-vaidya/cortx-re"),
-                                        string(name: 'CORTX_RE_BRANCH', value: "rocky-linux-custom-ci"),
+                                        string(name: 'CORTX_RE_BRANCH', value: "nightly-rgw-build"),
```

Comment:
@shailesh-vaidya, lets switch it to main branch and repo url right? so we can test it manually and confirm